.github/workflows/ci-macos.yml

```
@@ -12,16 +12,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: homebrew outdated pkg-config fix
-        run: |
-          brew update
-          brew upgrade
-          brew install pkgconf
-          # work around https://github.com/actions/runner-images/issues/10984
-
       - name: Install system dependencies
         run: |
-          brew install gnu-time opam gtksourceview3 adwaita-icon-theme expat libxml2
+          brew unlink pkg-config
+          brew install gnu-time opam gtksourceview3 adwaita-icon-theme expat libxml2 pkgconf
+          brew unlink pkgconf
+          brew link pkg-config
```

```suggestion
          brew install gnu-time opam gtksourceview3 adwaita-icon-theme expat libxml2
```
This should no longer be needed, the workers have been updated. In the logs you see it already:
```
 brew unlink pkg-config
  brew install gnu-time opam gtksourceview3 adwaita-icon-theme expat libxml2 pkgconf
  brew unlink pkgconf
  brew link pkg-config
  shell: /bin/bash -e {0}
Unlinking /usr/local/Cellar/pkgconf/2.3.0_1... 15 symlinks removed.
Warning: pkgconf 2.3.0_1 is already installed, it's just not linked.
To link this version, run:
  brew link pkgconf
```

---

we can try